deploy/staging/data-norge-frontend/data-norge-frontend-env.yaml

```diff
@@ -73,3 +73,13 @@ spec:
                             secretKeyRef:
                                 name: commonurl-staging
                                 key: ACCESS_REQUEST_API_HOST
+                      - name: FDK_DATASET_PREVIEW_API_KEY
+                        valueFrom:
+                            secretKeyRef:
+                                name: commonurl-staging
+                                key: FDK_DATASET_PREVIEW_API_KEY
```

Comment:
`secretKeyRef:
    name: fdk-dataset-preview-service
    key: API_KEY`

Reply:
Flyttet nøkkelen fra en fellessecret til en egen for preview-service.
Feilen din er _egentlig_ at du har skrevet commonurl-<namespace> istedet for common-<namespace> , men det er bedre at denne nøkkelen ligger i sin egen secret